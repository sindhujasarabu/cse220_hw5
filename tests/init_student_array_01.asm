.data
_num_students: .word 3
_id_list: .word 3126375 1264356 553656
_credits_list: .word 314 511 290
_names: .ascii "Kevin T. McDonnell\0Wolfie Seawolf\0Kelly Chen\0"

.align 2
_records: .space 24  # Must be 8 * num_students

.include "p3_tester.asm"
