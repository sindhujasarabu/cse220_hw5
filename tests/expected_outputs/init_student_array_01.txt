3126375 314 Kevin T. McDonnell
1264356 511 Wolfie Seawolf
553656 290 Kelly Chen
